### PR TITLE
All FAQs given

### PR DESCRIPTION
**Pull request by surma**

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -146,6 +146,7 @@ module.exports = function(eleventyConfig) {
     if (set.has(url)) return '';
     set.add(url);
 
+    console.log('emitting', url, 'on', page.url);
```
Comment (surma) [inline .eleventy.js]: :roll_eyes:

**Comment by jakearchibald** [comment]
The squiggle should be centered relative to other content in that column. It could also be a bit bigger at larger widths, maybe a % of the space?

The squiggle is also missing its shadow. Here it is with the shadow, and more optimised:

```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 423 414"><circle cx="103" cy="103" r="103" fill="#1a73e8" transform="translate(109 42)"/><path fill="#202124" d="M203 159a27 27 0 0 0-4-1l-16 1a76 76 0 0 0-11 1 22 22 0 0 1-3 0h-5l-3 1-12-1h-1l-14 1h-4a8 8 0 0 0-2 0l-10 1a22 22 0 0 1-3 0c-3 0-3 1-4 3a6 6 0 0 0 0 1 24 24 0 0 1 0 8l-2 9-4 9-7 11-2 1-4 3-1 1-9 5a5 5 0 0 0-3 2 60 60 0 0 1-4 6 15 15 0 0 1-3 2l-1 1c-3 3-6 3-9 5a12 12 0 0 1-6 1l-17 1a16 16 0 0 1-5-1l-10-4a16 16 0 0 1-4-2c-4-3-8-7-10-12a14 14 0 0 0-1-2l-1-2 2-3c1-1 3 0 3 1l5 7a23 23 0 0 0 13 9l5 1a27 27 0 0 0 12 1 17 17 0 0 1 3 0 16 16 0 0 0 8-2l8-4 3-4-6-2-4-1c-4-1-7-4-9-9a19 19 0 0 1-3-10 14 14 0 0 1 8-12l8-1a29 29 0 0 1 7 2 16 16 0 0 1 3 2l1 1c5 2 7 8 8 13v6a5 5 0 0 0 4-1l6-5 5-9c3-5 4-12 5-18a39 39 0 0 0 0-6 2 2 0 0 0-1-2l-9 1H80a32 32 0 0 1-4 0H65a7 7 0 0 0-3 0 9 9 0 0 1-4 0 71 71 0 0 0-8 0 13 13 0 0 1-4 0 38 38 0 0 0-11-2l-9-2-3-1a30 30 0 0 1-11-6 23 23 0 0 1-5-6 66 66 0 0 1-5-10l-1-6-1-11v-8l4-9a47 47 0 0 0 3-4 30 30 0 0 1 2-2l1-2c3-4 7-6 11-8l6-1h7c8-1 15 1 22 5l5 3 10-11a15 15 0 0 1 7-5l15-4 16-1c8 0 15 3 22 6a16 16 0 0 1 5 4l14 18 2 3 8-4a20 20 0 0 1 10-3h12l1 1c4-1 7 2 10 3a14 14 0 0 1 5 4l8 11a16 16 0 0 1 1 4 71 71 0 0 1 3 13l1 3v15h11a41 41 0 0 1 5-1l11-1h17a720 720 0 0 1 23 0 40 40 0 0 1 7 2 8 8 0 0 1 2 1 50 50 0 0 0 5 4 5 5 0 0 1 3 3c1 5 3 9 3 14v10a2 2 0 0 0 2 2 8 8 0 0 0 1 0l10-1 9 1h4a22 22 0 0 1 6 0 5 5 0 0 0 3 0 11 11 0 0 1 8 0 20 20 0 0 0 8 1 120 120 0 0 0 14-2 26 26 0 0 0 14-7 19 19 0 0 0 5-10l2-9a38 38 0 0 0 1-11v-16c0-2-1-2-3-2l-8 1-11 1a7 7 0 0 1-2 0 12 12 0 0 0-6 0h-4v6c0 4-1 5-5 5h-6l-5 1-9-1h-4c-2 0-3-1-3-3v-6a34 34 0 0 0-4-1l-7-1a13 13 0 0 0-3 0 28 28 0 0 1-5 0l-8 1-11-1a9 9 0 0 0-3 0l-10 2a19 19 0 0 1-14-4l-2-2a9 9 0 0 1-3-12l1-2a11 11 0 0 0-2-1l-6-2-2-1c-1-3-3-6-2-10s4-7 6-10l-6-4a9 9 0 0 1-3-10 14 14 0 0 0 1-2 5 5 0 0 1 2-4l6-4a5 5 0 0 0 1-1 9 9 0 0 0-1-2c-2-3-1-7 1-10 1-3 3-4 6-5l5-2a10 10 0 0 1 2-1l11-2 12-1 6-1v-8a124 124 0 0 1-2-18l2-10c1-2 4-3 7-3 7 2 11 6 14 13s5 15 6 23v4a3 3 0 0 1-1 4 3 3 0 0 1-3-2 32 32 0 0 1-2-5l-3-15a27 27 0 0 0-5-11l-6-4c-1-1-2-1-3 1v12l3 21a12 12 0 0 1 0 2c1 2-1 4-4 3h-7l-13 1a25 25 0 0 0-13 3 27 27 0 0 1-4 2 6 6 0 0 0-3 3c-1 3 0 4 2 4a19 19 0 0 0 4 0l12-2c5 0 10 1 14 3a7 7 0 0 1 4 5c0 3-1 5-4 6a5 5 0 0 1-1 0h-16a8 8 0 0 1-2 0l-9-3a6 6 0 0 0-3 0 20 20 0 0 0-9 5 2 2 0 0 0 0 4c4 3 9 6 14 6a33 33 0 0 0 4 0l4-1a21 21 0 0 0 4-1c5-2 10-2 15 1a45 45 0 0 0 5 2c2 3 1 5-2 6a28 28 0 0 1-8 3 25 25 0 0 1-6 0h-7a1 1 0 0 1-1 0l-9-1h-1c-2 2-6 1-8 4a4 4 0 0 0 2 7 62 62 0 0 0 9 3 6 6 0 0 0 4-1c4-3 8-3 13-4a47 47 0 0 0 9-1h7a4 4 0 0 1 0 5 36 36 0 0 1-3 4 13 13 0 0 1-9 4l-5 2a10 10 0 0 1-3 0h-10c-2-1-5 1-5 2a3 3 0 0 0 1 3l6 4h7l8-1a1 1 0 0 1 1 0l9 1a9 9 0 0 0 4 0 10 10 0 0 1 6-1 8 8 0 0 0 3 0 24 24 0 0 1 8 1h11v-3l-1-16c0-2 1-3 3-3h4l9-1a11 11 0 0 0 6-2 7 7 0 0 1 2-2c4-1 7-5 5-12a6 6 0 0 0-1 1l-10 7a13 13 0 0 1-11-1c-7-3-9-9-9-15a7 7 0 0 1 4-7 40 40 0 0 1 5-2l5-2a13 13 0 0 1 3 0h10a5 5 0 0 0 0-2l1-7c0-2-1-2-3-2h-10c-5 0-6-1-7-6a32 32 0 0 1 0-10v-8c0-3 1-4 4-4l9-1a46 46 0 0 1 7 0l5 1c2 1 1 3 1 5a30 30 0 0 1 1 5l1 10a2 2 0 0 0 2 2h5a5 5 0 0 1 3-1h12l6 1c4 0 5 2 7 5l4 11v19l1 16v12l1 6a6 6 0 0 0 1 0l18-1h8a4 4 0 0 1 3 3c1 1 0 3-1 4a8 8 0 0 1-3 1h-16l-4 1h-4c-2 0-2 1-2 3v22c0 6-2 11-3 17l-3 6a18 18 0 0 1-2 5 52 52 0 0 1-5 5h-2c-2 3-7 4-10 5a5 5 0 0 1-2 1l-10 1a6 6 0 0 0-1 0c-5 1-10 0-15-1a6 6 0 0 0-4 0 10 10 0 0 1-7 1 4 4 0 0 0-2-1h-15l-2 1c-4-2-8 0-12 0v8l-1 22-1 11v31a5 5 0 0 1-7 4 16 16 0 0 1-4-3l-10-8a12 12 0 0 1-3-2l-3-3a71 71 0 0 0-10 9 11 11 0 0 1-1 2l-8 8c-1 2-5 3-7 2a3 3 0 0 1-2-3l1-14v-33l1-12a47 47 0 0 0 0-6v-7c1-4 2-8 7-9a23 23 0 0 0 4-2l12-3h17l7-1v-13a15 15 0 0 0-2-8 9 9 0 0 0-4-4 36 36 0 0 0-13-2 149 149 0 0 1-18 0 16 16 0 0 0-2 0l-16 1a179 179 0 0 0-20 2 39 39 0 0 1-4 0 70 70 0 0 0 0 16v11a11 11 0 0 0 0 3l1 3-1 27c-1 9-1 19-3 28a57 57 0 0 1-5 15 45 45 0 0 1-5 8l-8 8a18 18 0 0 1-3 2l-4 3a7 7 0 0 1-1 1l-10 3-3 1-3 1c-8 1-16 2-23 0-14-4-25-12-31-25l-4-19a37 37 0 0 1 2-19c2-4 3-8 7-11l3-4 5-5a104 104 0 0 1 11-5l9-3a36 36 0 0 1 14-1 19 19 0 0 0 2 0 29 29 0 0 1 15 3h2l5 3a81 81 0 0 1 9 8l4 7a3 3 0 0 1-2 5 4 4 0 0 1-5-1 20 20 0 0 1-1-3c-1-3-3-6-5-7a14 14 0 0 1-3-3c-2-2-3-3-6-3a6 6 0 0 1-3-1c-1-2-3-2-5-2l-12-1c-4 0-8 1-11 3a1 1 0 0 1-1 1c-3 0-6 2-9 4a23 23 0 0 0-6 4 36 36 0 0 0-5 6 33 33 0 0 0-3 7c-3 8-4 15-2 23l2 9c2 6 5 11 10 15a52 52 0 0 0 12 7c6 3 12 4 19 3l7-2a6 6 0 0 1 2-1c4 0 6-1 9-3 6-3 10-7 14-12a31 31 0 0 0 6-11 87 87 0 0 0 4-23l1-17a12 12 0 0 0 0-4v-15l1-11-1-12v-13zm-99-4v-4l1-19a15 15 0 0 0-2-6l-1 1-4 7c-2 2-4 3-7 2-2 0-3-1-2-3l2-3 4-6 6-12a4 4 0 0 1 3-2 10 10 0 0 1 10 4 16 16 0 0 0 2 4l11 10a4 4 0 0 1 1 4l-3 1c-4 1-6-1-8-3l-4-4v27c0 2 0 3 2 2l12-1h31l18-1 11-1h13c2 0 3-1 3-3v-9a97 97 0 0 0-1-10c0-4 0-7-2-10-1-4-4-7-6-11a15 15 0 0 0-5-4 27 27 0 0 0-14-4c-3 0-5 0-8 2-4 2-8 5-10 10-1 3-4 3-6 3s-4-1-4-3c-1-5-4-9-6-14l-6-8a27 27 0 0 0-12-10 48 48 0 0 0-6-2 20 20 0 0 0-5-1h-10l-8 1-11 3a10 10 0 0 0-3 3c-5 4-10 9-13 15a5 5 0 0 0 0 2c0 3-1 5-4 5-4 2-6 0-6-5a8 8 0 0 0-1-4 20 20 0 0 0-6-5 24 24 0 0 0-16-2 20 20 0 0 1-4 0c-3 0-5 0-7 2a5 5 0 0 1-1 1c-5 3-8 7-11 12a18 18 0 0 0-2 8l-1 12c1 4 1 9 3 13l6 7c5 5 12 7 19 8h1c1 2 4 1 6 2a16 16 0 0 0 4 0 320 320 0 0 0 14 0h32l7-1h4zm280-35a18 18 0 0 0 1-2l-1-18-1-16V66c0-4-2-7-4-10a3 3 0 0 0-3-2l-7-1a25 25 0 0 0-6 1l-5 1h-6a11 11 0 0 0 0 2l-1 13a9 9 0 0 0 0 3l3 8c2 4 1 9 0 13a8 8 0 0 1-2 3 27 27 0 0 1-12 9h-2l-9 2c-2 0-3 0-3 2l1 13 15-1v-4l-1-4a2 2 0 0 1 3-2h1c3 0 4 0 4 3l1 5c0 3 1 3 3 3a26 26 0 0 1 3-1h11l17-2zm-94 69h-20a8 8 0 0 0-2 0l-12 3c-3 1-5 3-5 6v12l-1 10-1 14v26l5-3a50 50 0 0 0 3-4 53 53 0 0 0 5-6c2-3 4-5 8-4a3 3 0 0 1 3 1c4 5 8 8 13 11l2 2a14 14 0 0 0 1-3v-20a21 21 0 0 1 0-3l1-19v-23zM75 202l-1-7c-1-2-2-6-6-7l-6-2a3 3 0 0 0-3 0c-2 2-3 6-2 10l2 4c2 3 4 5 8 6a26 26 0 0 1 3 1c3 1 5 0 5-3v-2zM337 71l-10 3c-3 1-3 2-3 5l1 2c0 3 3 5 5 7a2 2 0 0 0 2 0c6-2 9-6 10-12a3 3 0 0 0 0-2 40 40 0 0 0-5-3zm7-23a84 84 0 0 0-1-13 4 4 0 0 0-1-2l-1-1c-3 0-6-1-9 1v12a7 7 0 0 0 1 3 2 2 0 0 0 2 2 25 25 0 0 0 9-2zm-17 87h15v-5a40 40 0 0 0-15 1zm-59-71v1h1v-1z"/><path fill="#f0f1f5" d="M419 414H6l67-11h278z"/></svg>
```

**Comment by jakearchibald** [comment]
At smaller widths the content gets really squeezed.

Screenshot 2019-09-24 at 15 21 15

Feels like we should drop to single column sooner.

**Comment by jakearchibald** [comment]
Can we make the blue headings not-blue? They look like links.

**Pull request merged**